skbio/io/format/embl.py

```diff
@@ -0,0 +1,1568 @@
+# info from Sequence and its derived objects Here is a dictionary of keys
+# conversion (EMBL->GB). All the unspecified keys will remain in embl format
+KEYS_TRANSLATOR = {
+                   # identification
+                   'ID': 'LOCUS',
+                   'AC': 'ACCESSION',
+                   # PA means PARENT ACCESSION (?) and applies to
+                   # feature-level-products entries
+                   'PA': 'PARENT_ACCESSION',
+                   'PR': 'PROJECT_IDENTIFIER',
+                   'DT': 'DATE',
+                   'DE': 'DEFINITION',
+                   'GN': 'GENE_NAME',  # uniprot specific
+                   'KW': 'KEYWORDS',
+                   # Source (taxonomy and classification)
+                   'OS': 'ORGANISM',
+                   'OC': 'taxonomy',
+                   'OG': 'organelle',
+                   # reference keys
+                   'RA': 'AUTHORS',
+                   'RP': 'REFERENCE',
+                   'RC': 'REFERENCE_COMMENT',
+                   'RX': 'CROSS_REFERENCE',
+                   'RG': 'GROUP',
+                   'RT': 'TITLE',
+                   'RL': 'JOURNAL',
+                   # Cross references
+                   'DR': 'DBSOURCE',
+                   'CC': 'COMMENT',
+                   # features
+                   'FH': 'FEATURES',
+                   'FT': 'FEATURES',
+                   'SQ': 'ORIGIN',
+                   }
+
+
+# the inverse of KEYS_TRANSLATOR, for semplicity
+REV_KEYS_TRANSLATOR = {v: k for k, v in KEYS_TRANSLATOR.items()}
+
+
+# the original genbank _yield_section divides entries in sections relying on
+# spaces (the same section has the same level of indentation). Uniprot entries
+# have a key for each line, so to divide record in sections I need to define a
+# correspondance for each key to section, then I will divide a record in
+# sections using these section name.
+KEYS_2_SECTIONS = {
+                   # identification
+                   'ID': 'LOCUS',
+                   'AC': 'ACCESSION',
+                   # PA means PARENT ACCESSION (?) and applies to
+                   # feature-level-products entries
+                   'PA': 'PARENT_ACCESSION',
+                   'PR': 'PROJECT_IDENTIFIER',
+                   'DT': 'DATE',
+                   'DE': 'DEFINITION',
+                   'GN': 'GENE_NAME',  # uniprot specific
+                   'KW': 'KEYWORDS',
+                   # Source (taxonomy and classification)
+                   'OS': 'SOURCE',
+                   'OC': 'SOURCE',
+                   'OG': 'SOURCE',
+                   # reference keys
+                   'RA': 'REFERENCE',
+                   'RP': 'REFERENCE',
+                   'RC': 'REFERENCE',
+                   'RX': 'REFERENCE',
+                   'RG': 'REFERENCE',
+                   'RT': 'REFERENCE',
+                   'RL': 'REFERENCE',
+                   'RN': 'SPACER',
```

Comment:
Yes, it's Reference Number. However, in order to split between reference with `_embl_yield_section`, I need to change this value to `spacer`. This dictionary is needed only to process group of keys with the same function. The reference number information will be stored as reference index in the final list of references. I wrote such things as comments